### ISSUE TITLE
Proper location / include mechanism for WeatherSensorCfg.h?

### ISSUE THREAD

**Issue by matthias-bs** [matthias-bs/BresserWeatherSensorReceiver#1]
`WeatherSensorCfg.h` should be modified for a specific sketch and therefore not be located in the library directory, but it has to be included both in the sketch and in the library.
Unfortunately, it is not found when the preprocessor tries to include it in the library (file `WeatherSensor.h`).
Any suggestions?

As a workaround, copy `WeatherSensorCfg.h` to the library's src/ directory, too.
Or copy all files to your sketch directory, e.g.:
```
BresserWeatherSensorMQTT.ino
secrets.h
WeatherSensorCfg.h
WeatherSensor.cpp
WeatherSensor.h
```

**Comment by matthias-bs** [comment]
Also see https://github.com/matthias-bs/BresserWeatherSensorReceiver/issues/4#issuecomment-1272290529

**Comment by matthias-bs** [comment]
I hope I can provide a clean solution now.

1. I have modified the library and the example [BresserWeatherSensorMQTT.ino](https://github.com/matthias-bs/BresserWeatherSensorReceiver/blob/main/examples/BresserWeatherSensorMQTT/BresserWeatherSensorMQTT.ino)

    This example now does not have a configuration file of its own any longer, but uses the one in BresserWeatherSensorReceiver/src. You can change it in your local Arduino/libraries/BresserWeatherSensorReceiver/src directory if you are always using the same setup (receiver type and wiring).

    I will change the other examples in the same way.

2. I have added a customized version in [BresserWeatherSensorMQTTCustom](https://github.com/matthias-bs/BresserWeatherSensorReceiver/tree/main/examples/BresserWeatherSensorMQTTCustom). Please check the [Readme.md](https://github.com/matthias-bs/BresserWeatherSensorReceiver/blob/main/examples/BresserWeatherSensorMQTTCustom/Readme.md).

    Following this structure, you are able to use a customized configuration per sketch.

**Comment by matthias-bs** [comment]
I assume this has been fixed in release [v0.4.0](https://github.com/matthias-bs/BresserWeatherSensorReceiver/releases/tag/v0.4.0).
